src/autorest-core/test/resources/small-input/generated/swagger

```diff
@@ -0,0 +1,1348 @@
+{
```

Comment:
whoops? looks like you've run autorest manually here, this file does not belong on github